app/controllers/AddressLookup.scala

```diff
@@ -25,7 +26,17 @@ trait AddressLookup extends PropertyLinkingController {
   val addresses = Wiring().addresses
 
   def findByPostcode(postcode: String) = Action.async { implicit request =>
-    addresses.findByPostcode(postcode) map { res => Ok(Json.toJson(res)) }
+    if (postcode.trim.contains(" ")) {
```

Comment:
the postcode lookup should accept partial postcodes e.g. EX1 - and these currently work on CI3